player/playloop.c

```diff
@@ -1289,6 +1289,8 @@ void run_playloop(struct MPContext *mpctx)
 
     mp_process_input(mpctx);
 
+    mp_option_run_callbacks(mpctx);
```

Comment:
So this in practice, delays option callback until next playloop tick. Would there be a problem with that delay?

Reply:
It's on the order of hundreds of milliseconds at most. I can't imagine that would be an issue.